app/src/main/java/com/muzima/model/AvailableForm.java

```diff
@@ -32,14 +33,7 @@ public void setDownloaded(boolean downloaded) {
     }
 
     public boolean isRegistrationForm(){
-        if(tags == null){
-            return false;
-        }
-        for (Tag tag : tags) {
-            if(REGISTRATION.equalsIgnoreCase(tag.getName())){
-                return true;
-            }
-        }
-        return false;
+        return Constants.FORM_DISCRIMINATOR_REGISTRATION.equalsIgnoreCase(getDiscriminator()) ||
```

Comment:
Hi @yoctopus , your commit includes changes already submitted in a separate pull request (MUZIMA-330). There will be a conflict when processing the other PR  if we merge this as it is. Could you revert non-copyright info changes? Thanks.